mainCtrl.js

```diff
@@ -0,0 +1 @@
+mainCtrl.js
```

Comment:
Expected an assignment or function call and instead saw an expression.<br>Missing semicolon.<br>'mainCtrl' is not defined.